packages/common/components/ColorTheme/styled/floatingButton.js

```diff
@@ -2,28 +2,40 @@ import styled, { css } from "styled-components";
 import {
   StyledCircleWrap,
   StyledFloatingButton,
+  IconBox,
+  StyledCircle,
 } from "@docspace/common/components/FloatingButton/StyledFloatingButton";
 import Base from "@docspace/components/themes/base";
 
-const getDefaultStyles = ({ $currentColorScheme, color, icon, theme }) =>
+const getDefaultStyles = ({
+  $currentColorScheme,
+  color,
+  icon,
+  theme,
+  displayProgress,
+}) =>
   $currentColorScheme &&
   css`
-    background: ${color
-      ? color
-      : theme.isBase
-      ? $currentColorScheme.accentColor
-      : icon === "upload"
-      ? theme.floatingButton.backgroundColor
-      : $currentColorScheme.accentColor} !important;
+    background: ${color ? color : $currentColorScheme.main.accent} !important;
```

Comment:
Simplify condition to ${color || $currentColorScheme.main.accent}

Reply:
Fix